changelog/_unreleased/2020-09-17-fix-product-media-position-duplication.md

```diff
@@ -0,0 +1,34 @@
+---
+title:         Fix bug when images are add to a product the position 1 can be assigned multiple times
+issue:         NEXT-9292
+author:        Joshua Behrens
+author_email:  code@joshua-behrens.de
+author_github: JoshuaBehrens
+___
+# Administration
+* Match procedure to assign `product_media.position` in `sw-product-media-form` and `sw-product-detail` so it is always set to prevent a bug where the position 1 is automatically assigned which results in unexpected sorting and selection behaviours
+___
+# Upgrade Information
```

Comment:
Upgrade information is meant for third party developers, if they have to change their code according to the new changes. 

I don't know if this really matches here. @Phil23 ?

Reply:
Oh I thought for any shop owners. Ambiguity 🎉

Reply:
Also the headline should be more topic related, because nobody will know, what "this bug" actually is.

Reply:
And the "Upgrade Information" should only describe, what developers have to change when upgrading to the specific version.

Reply:
Is there any other place where I should place the SQL? I mean there were upgrade guides for agencies/shop owners in an update for shopware 5. I'd like to keep it there so people are aware of this.

Reply:
You mean the headline is not topic specific when it is merged into the compiled markdown?